e2e-tests/demand-backup-sharded/run

```diff
@@ -25,6 +25,15 @@ check_data() {
 	done
 }
 
+check_exported_mongos_service_endpoint() {
+	local host=$1
+
+	if [ "$host" != "$(kubectl_bin get psmdb $cluster -o=jsonpath='{.status.host}')" ]; then
```

Comment:
was this suppose to be host or ip? because IIRC we should check that ip didn't change, but maybe I misunderstood something.

Reply:
@tplavcic should we test mongos service per pod or just endpoint?

Reply:
I think for mongos we only have endpoint, since there is only one service.